airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/SecretsRepositoryReaderTest.java

```diff
@@ -116,4 +122,111 @@ void testDumpConfigsWithSecrets() throws IOException {
     assertEquals(expected, actual);
   }
 
+  @Test
+  public void testReadingServiceAccount() throws JsonValidationException, ConfigNotFoundException, IOException {
+    final ConfigRepository configRepository = mock(ConfigRepository.class);
+    final SecretPersistence secretPersistence = mock(SecretPersistence.class);
+    final RealSecretsHydrator realSecretsHydrator = new RealSecretsHydrator(secretPersistence);
+    final SecretsRepositoryReader secretsRepositoryReader =
+        spy(new SecretsRepositoryReader(configRepository, realSecretsHydrator));
+
+    final UUID workspaceId = UUID.fromString("13fb9a84-6bfa-4801-8f5e-ce717677babf");
```

Comment:
Many of the UUIDs and constants are reused in the two test files, like this workspace ID, the service account ID, and HMAC key. They can be moved to `MockData.java` for clarity.